src/analyserexternalvariable.cpp

```diff
@@ -16,9 +16,8 @@ limitations under the License.
 
 #include "libcellml/analyserexternalvariable.h"
 
-#include "libcellml/component.h"
-
 #include "analyserexternalvariable_p.h"
+#include "commonutils.h"
```

Comment:
Added for what reason?

Reply:
As a result of some cleaning up (see 57289480).